Libraries/Text/RCTText.m

```diff
@@ -237,7 +237,11 @@ - (BOOL)canBecomeFirstResponder
 - (BOOL)canPerformAction:(SEL)action withSender:(id)sender
 {
   if (action == @selector(copy:)) {
```

Comment:
if there is a nextResponser, shouldn't it also be able to determine if it returns YES?